templates/addons/calico.yaml

```diff
@@ -10,13 +10,13 @@ data:
   # Typha is disabled.
   typha_service_name: "none"
   # Configure the backend to use.
-  calico_backend: "vxlan"
-  # On Azure, the underlying network has an MTU of 1400, even though the network interface will have an MTU of 1500.
-  # We set this value to 1350 for “physical network MTU size minus 50” since we use VXLAN, which uses a 50-byte header.
-  # If enabling Wireguard, this value should be changed to 1340 (Wireguard uses a 60-byte header).
-  # https://docs.projectcalico.org/networking/mtu#determine-mtu-size
-  veth_mtu: "1350"
-  
+  calico_backend: "bird"
+
+  # Configure the MTU to use for workload interfaces and tunnels.
+  # By default, MTU is auto-detected, and explicitly setting this field should not be required.
+  # You can override auto-detection by providing a non-zero value.
+  veth_mtu: "0"
```

Comment:
if we do go ahead with this change we still need the veth_mtu update (see #1353 and the comment above)